src/api/sfc-spec.md

```diff
@@ -109,7 +109,7 @@ Note that integration with various pre-processors may differ by toolchain. Check
 - [Vue CLI](https://cli.vuejs.org/guide/css.html#pre-processors)
 - [webpack + vue-loader](https://vue-loader.vuejs.org/guide/pre-processors.html#using-pre-processors)
 
-## Src Imports {#src-imports}
+## src Imports {#src-imports}
```

Comment:
If we're using lowercase for `src` because it's code, should we maybe wrap it in backticks too?

```suggestion
## `src` Imports {#src-imports}
```